wgsl/index.bs

```diff
@@ -2106,20 +2118,8 @@ equality_expression
         OpINotEqual
         OpFOrdNotEqual
 
-and_expression
-  : equality_expression
-  | and_expression AND equality_expression
-
-exclusive_or_expression
-  : and_expression
-  | exclusive_or_expression XOR and_expression
-
-inclusive_or_expression
-  : exclusive_or_expression
-  | inclusive_or_expression OR exclusive_or_expression
-
 short_circuit_and_expression
-  : inclusive_or_expression
+  : equality_expression
   | short_circuit_and_expression AND_AND inclusive_or_expression
```

Comment:
This needs to be updated as well

Reply:
@dj2 Does the latest commit address concerns?